modules/localization/msf/local_tool/data_extraction/cyber_record_reader.cc

```diff
@@ -36,13 +36,15 @@ void CyberRecordReader::Subscribe(
   topics_.push_back(topic);
 }
 
-void CyberRecordReader::Read(const std::string &file_name) {
-  RecordReader reader(file_name);
-  cyber::record::RecordMessage message;
-  while (reader.ReadMessage(&message)) {
-    auto itr = call_back_map_.find(message.channel_name);
-    if (itr != call_back_map_.end()) {
-      itr->second(message.content);
+void CyberRecordReader::Read(const std::vector<std::string> &file_names) {
```

Comment:
It is recommended to keep the original interface and extend it by overloading.

Reply:
I restored the original interface